src/jit/lsra.cpp

```diff
@@ -8270,7 +8270,7 @@ void LinearScan::resolveRegisters()
             {
                 JITDUMP(" internal");
                 GenTreePtr indNode = nullptr;
-                if (treeNode->OperIsIndir())
+                if (treeNode->OperGet() == GT_IND)
```

Comment:
@sivarv I had done so prior to those changes, but obviously missed this one. I have just done so again. There are some places in instr.cpp where the check is broader than it needs to be (e.g. in `instEmit_indCall()` it checks `addr->OperIsIndir()` on the call address, which should probably just be a check for `GT_IND`, but it is not actually incorrect, as that is the only indirection we will encounter there).